### DISCUSSION TITLE
Community meeting on 21st Nov

### DISCUSSION THREAD

**Discussion by pavithraes** [nebari-dev/nebari#2123] [Community meetings]
We'll have a community meeting today, please add discussion topics and updates to the meeting notes!

🕒 Meeting date & time: 3-4 pm GMT

💻 Meeting room: meet.google.com/rio-ngrj-kyh

📝 Meeting notes: https://hackmd.io/@nebari/Hkc3yEc4a

🗓️ Meeting calendar: nebari.dev/docs/community#community-meetings

**Comment by pavithraes** [comment]
Pinging @davidchapuis - I noticed you were interested in the previous meeting, and I'm sorry you couldn't join. I'll keep an eye out for joining requests today. :)

**Reply by davidchapuis** [reply]
Hi @pavithraes thanks, that's kind of you :)
Actually won't be able to join today but hope to join the next one
